### ISSUE TITLE
timestamp and datetime for order structure are inverted.

### ISSUE THREAD

**Issue by tumma72** [ccxt/ccxt#17155]
### Operating System

MacOS

### Programming Languages

python

### CCXT Version

2.9.15

### Description

It appears that the datetime and timestamp fields have been swapped in the Order Structure in the last 2 versions 2.9.14, and 2.9.15. I have tested it with bittrex, kraken, binance and cryptocom exchange, all the same.

Here is a printout of the order structure received from the exchange:
```
{'info': {'symbol': 'BTCUSDT', 'orderId': '19098998736', 'orderListId': '-1', 'clientOrderId': 'x-R4BD3S82e7872c207c7f0f0e8662e0', 'price': '23297.45000000', 'origQty': '0.00045000', 'executedQty': '0.00000000', 'cummulativeQuoteQty': '0.00000000', 'status': 'CANCELED', 'timeInForce': 'GTC', 'type': 'LIMIT', 'side': 'BUY', 'stopPrice': '0.00000000', 'icebergQty': '0.00000000', 'time': '1677146586081', 'updateTime': '1677146587166', 'isWorking': True, 'workingTime': '1677146586081', 'origQuoteOrderQty': '0.00000000', 'selfTradePreventionMode': 'NONE'}, 'id': '19098998736', 'clientOrderId': 'x-R4BD3S82e7872c207c7f0f0e8662e0', 'timestamp': '2023-02-23T10:03:06.081Z', 'datetime': 1677146586081, 'lastTradeTimestamp': None, 'symbol': 'BTC/USDT', 'type': 'limit', 'timeInForce': 'GTC', 'postOnly': False, 'reduceOnly': None, 'side': 'buy', 'price': 23297.45, 'triggerPrice': None, 'amount': 0.00045, 'cost': 0.0, 'average': None, 'filled': 0.0, 'remaining': 0.00045, 'status': 'canceled', 'fee': {'currency': None, 'cost': None, 'rate': None}, 'trades': [], 'fees': [{'currency': None, 'cost': None, 'rate': None}], 'stopPrice': None}
```
You can see that `'timestamp':  '2023-02-23T10:03:06.081Z'` while `'datetime': 1677146586081` is in fact the timestamp in milliseconds. I am fairly sure this happens in one of the last 2-3 releases because I have Pydantic validation tests in place to ensure the input to my software is according to specification.

### Code

Here is a small snippet to test it:
```
orders: List[Dict[str, Any]] = await ccxt_exchange.fetch_orders(
            "BTC/USDT", limit=1
        )
        if orders is not None and isinstance(orders, list) and len(orders) > 0:
            ccxt_order: Dict[str, Any] = orders[0]
            print(f"Order from exchange: {ccxt_order}")
```

**Comment by carlosmiei** [comment]
Hello @tumma72, thanks for letting us know. Probably some merge conflict caused this issue, but we will fix it asap.

**Comment by carlosmiei** [comment]
@tumma72 it should be fixed now (from version 2.9.16)

Thanks once again.

**Comment by tumma72** [comment]
I can confirm it is fixed in 2.9.16, thanks 😉
